package.json

```diff
@@ -28,10 +28,12 @@
     "crypto-js": "3.1.9-1",
     "electron-updater": "2.21.10",
     "ethereum-blockies-base64": "1.0.2",
-    "ethereumjs-abi": "git://github.com/ethereumjs/ethereumjs-abi.git#09c3c48fd3bed143df7fa8f36f6f164205e23796",
+    "ethereumjs-abi":
```

Comment:
Did you intentionally add a new line for each of these?

Reply:
We seem to have some weird formatter clashes with package.json. Whenever I edit it, it often toggles hard wrapping these key/value pairs on/off. I have a feeling that Prettier turns it on and yarn turns it off.

Reply:
It seems that we have 4 different configs for Prettier (tests, formatAll, travis, and VSC).
We need to bring it down to a single one.

Reply:
All those scripts and tools should be using the same Prettier config: https://github.com/MyCryptoHQ/MyCrypto/blob/develop/.prettierrc

I think the issue is with Yarn, because it has its own JSON formatter that doesn't use Prettier. I agree that we should make it consistent, but I'm not sure what the best approach is.